caffe2/CMakeLists.txt

```diff
@@ -1259,6 +1260,7 @@ install(FILES
   "${TORCH_SRC_DIR}/extension.h"
   "${TORCH_SRC_DIR}/custom_class.h"
   "${TORCH_SRC_DIR}/library.h"
+  "${TORCH_SRC_DIR}/library_stable.h"
```

Comment:
You most likely also need to change "buckbuild.bzl" and friends. Looking for "library.h" and pattern matching should work.
Also importing internally just to check might be helpful to avoid noisy revert